### PR TITLE
Add changelog for 3.19.0

### PR DESCRIPTION
**Pull request by mariusSincovici**

### CODE REVIEW

**Review by Savolro — APPROVED** [review]

File: contrib/changelog/prod/3.19.0_1726656718.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* As quantum computing advances, so do our measures to protect your data. You can now enable post-quantum encryption against the cyber threats of the future. It’s always better to be prepared than to wish you were. Or, as they say: "Be quantum-proof." Use the command: nordvpn set PQ <enable>.
```
Comment (keliramu) [inline contrib/changelog/prod/3.19.0_1726656718.md:1]: $ nordvpn set PQ on
Command 'PQ' doesn't exist.

$ nordvpn set pq on
Post-quantum VPN is set to 'enabled' successfully.

Comment (keliramu) [reply contrib/changelog/prod/3.19.0_1726656718.md:1]: $ nordvpn set PQ <enable>
bash: syntax error near unexpected token `newline'

Comment (keliramu) [reply contrib/changelog/prod/3.19.0_1726656718.md:1]: Misleading recommendation what command and how should be used

File: contrib/changelog/prod/3.19.0_1726656718.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+* Now even more devices can run NordVPN. We’ve added support for ARM64 architecture in the Snap package.
```
Comment (keliramu) [inline contrib/changelog/prod/3.19.0_1726656718.md:2]: We added ARM64 and ARM32 (armhf) Snap packages

**Review by devzbysiu — APPROVED** [review]

**Review by keliramu — APPROVED** [review]

**Pull request merged**